src/components/map/MapService.js

```diff
@@ -370,6 +370,19 @@ goog.require('ga_urlutils_service');
           layers: params.LAYERS,
           format: 'image/png',
           service: 'WMS',
+          version: '1.1.1',
+          request: 'GetMap',
+          srs: 'EPSG:4326',
+          bbox: '{westProjected},{southProjected},' +
+                '{eastProjected},{northProjected}',
+          width: '256',
+          height: '256',
+          styles: 'default',
+          transparent: 'true'
+        } || {
+          layers: params.LAYERS,
+          format: 'image/png',
+          service: 'WMS',
           version: '1.3.0',
           request: 'GetMap',
           crs: 'CRS:84',
```

Comment:
but it always accepts only '1.3.0' as its version, because this is set as default version[here](https://github.com/geoadmin/mf-geoadmin3/blob/kom_wms2/src/js/ImportWmsController.js#L9)
Should I change something there?